fix(Forms): provide `connectWithPath` in the change validator and onBlurValidator (#3895)
dnbexperience/eufemia: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/getting-started.mdx

<Field.PhoneNumber schema={schema} />
```

#### onBlurValidator and validator

> The `onBlurValidator` and `validator` accepts a function that takes the current value of the field as an argument and returns an error message if the value is invalid:

Reviewer: ```suggestion
The `onBlurValidator` and `validator` properties accepts a function that takes the current value of the field as an argument and returns an error message if the value is invalid:
```

You can find more info about error messages in the [Error messages](/uilib/extensions/forms/Form/error-messages/) docs.

##### Connect with another field

> You can also use the `connectWithPath` function to connect the validator to another field. This allows you to rerender the validator function once the value of the connected field changes:

Reviewer: ```suggestion
You can also use the `connectWithPath` function to connect the validator to another field. This allows you to render the validation function when the value of the connected field changes:
```

```tsx
import { Form, Field } from '@dnb/eufemia/extensions/forms'

const validator = (value, { connectWithPath }) => {
  const { getValue } = connectWithPath('/myReference')
  const amount = getValue()
  if (amount >= value) {
    return new Error(`The amount should be greater than ${amount}`)
  }
}

render(
  <Form.Handler>

>     <Field.Number path="/myRefference" defaultValue={2} />

Reviewer: ```suggestion
    <Field.Number path="/myReference" defaultValue={2} />
```

    <Field.Number
      path="/withValidator"
      defaultValue={2}

>       validator={validator} // NB: You may use "onBlurValidator"

Reviewer: ```suggestion
      validator={validator} // NB: You may use "onBlurValidator" depending on your use case.
```

      //
    />
  </Form.Handler>,
)
```

> By default, the validator function will only run when the "/withValidator" field is changed (and blurred). When the error message is shown, it will update the error message with the new value of the "/myRefference" field.

Reviewer: ```suggestion
By default, the validator function will only run when the "/withValidator" field is changed (and blurred). When the error message is shown, it will update the error message with the new value of the "/myReference" field.
```

You can also change this behaviour by using the following props:

> - `validateInitially` Will run the validator initially

Reviewer: ```suggestion
- `validateInitially` Will run the validator initially.
```

> - `continuousValidation` Will run the validator on every change, including when the connected field changes

Reviewer: ```suggestion
- `continuousValidation` Will run the validator on every change, including when the connected field changes.
```

> - `validateUnchanged` Is a combination of validateInitially and continuousValidation

Reviewer: ```suggestion
- `validateUnchanged` Is a combination of `validateInitially` and `continuousValidation`.
```
